api/envoy/extensions/filters/http/aws_lambda/v3/aws_lambda.proto

```diff
@@ -62,6 +62,23 @@ message Config {
   // the provider chain is limited to the AWS credentials file Provider.
   // Other providers are ignored
   string credentials_profile = 5;
+
+  // Specifies the credentials to be used. This parameter is optional.
+  // If set, it will override other providers and takes precedence, the provider chain is limited
+  // to the configuration credentials provider. Other providers are ignored
```

Comment:
Please clarify what other providers, and what's the precedence if `credentials_profile` is supplied as well?

Reply:
when I mention providers is the providers chain following this order https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/aws_lambda_filter#credentials

and explicit credentials should have preference over credentials_profile. I have added test for that but I agree it is not clear, I will try to explain better here. And probably Adding the description in the doc (link before) could be helpful.